pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -64,7 +64,7 @@ def restore_weights(self, model: LightningModule) -> None:
             rank_zero_info(f'restored hpc model from: {checkpoint_path}')
 
         # 2. Attempt to restore states from `resume_from_checkpoint` file
-        elif self.trainer.resume_from_checkpoint is not None and not self.trainer.testing:
+        elif self.trainer.resume_from_checkpoint is not None:
```

Comment:
why is this gone? thomas just added it in #5161

Reply:
`self.trainer.checkpoint_connector.restore_weights(model)` will be called during training only.. so this won't be called while testing or evaluating. You can check the call to `setup_training` in the changes.